### DISCUSSION TITLE
How to reset MPD from command line

### DISCUSSION THREAD

**Discussion by jackjhill** [rern/rAudio#1271] [General]
Hello - my system seems to be stuck in a library update loop - doesn't appear to be updating library (has been at it for 24hrs) but I have no means to stop the update, or reset.  Rebooting hasn't fixed.  In the mean time it is reporting no entries in the library, so am pretty stuck

Is there a way to stop and restart the MPD so that I can start afresh?

Regards

Jack

**Comment by cmh714** [comment]
Kill mpd with  systemctl stop mpd.service
delete the db at /srv/http/data/mpd/mpd.db
and restart mpd or reboot.

I think thats what your asking, and should work fine. :)

**Reply by skycamel** [reply]
systemctl restart mpd

as simple as that (I use this sometimes when my DAB somehow plays up, no reboot needed)
Mind you, this can sometimes take up to a minute before it actually will execute the command, especially as your cpu runs close to 100%

**Reply by cmh714** [reply]
except he has a db problem, which a restart wouldnt solve....

**Comment by jackjhill** [comment]
Much thanks - will give that a go.
